server/config/buildTables.js

```diff
@@ -1,14 +1,18 @@
 import pool from '.';
 
+const { log } = console;
+
 const userTableQuery = `
   CREATE TABLE IF NOT EXISTS users(
     id SERIAL PRIMARY KEY,
-    first_name VARCHAR(100) NOT NULL,
-    last_name VARCHAR(100) NOT NULL,
-    user_name VARCHAR(100) NOT NULL,
+    username VARCHAR(100) UNIQUE NOT NULL,
+    firstname VARCHAR(100) NOT NULL,
+    lastname VARCHAR(100) NOT NULL,
```

Comment:
I'm not sure we decided on a naming convention for our db columns but I think underscores are more readable for serperating words. Are there any particular reasons you chose this?

Reply:
@Nerocodes Did you see the dB diagram we created? I believe we all agreed on that at the beginning of the sprint.

Reply:
Alright!